### PR TITLE
Added r-blaster recipe

### PR DESCRIPTION
**Pull request by jeevannavar**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
_The other GitHub user listed in the maintainer section is the author of the R package and is willing to be listed here._
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by manutamminen** [comment]
I can be listed as a maintainer.

File: recipes/r-blaster/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    - r-rcpp >=1.0.5
+
+test:
+  commands:
+    - $R -e "library('blaster')"           # [not win]
+    - "\"%R%\" -e \"library('blaster')\""  # [win]
+
+about:
+  home: https://github.com/manutamminen/blaster
+  license: BSD_3_clause
```
Comment (cbrueffer) [inline recipes/r-blaster/meta.yaml:49]:
```suggestion
  license: BSD-3-Clause
```

File: recipes/r-blaster/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    C++11 and using native R datatypes. Blaster is based on 'nsearch' - Schmid et al
+    2018; <doi:10.1101/399782>.
+  license_family: BSD
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/BSD_3_clause'
+    - LICENSE
+
+extra:
+  recipe-maintainers:
+    - manutamminen
```
Comment (cbrueffer) [inline recipes/r-blaster/meta.yaml:60]: Please add `conda-forge/r` as maintainer.

**Review by cbrueffer — CHANGES_REQUESTED** [review]
Looks good; please fix the two issues I raised and then this is good to go.

**Comment by jeevannavar** [comment]
> Looks good; please fix the two issues I raised and then this is good to go.

I have fixed both the issues.

**Pull request merged**